x-pack/plugins/data_visualizer/public/application/common/components/stats_table/data_visualizer_stats_table.tsx

```diff
@@ -221,7 +223,7 @@ export const DataVisualizerTable = <T extends DataVisualizerTableItem>({
           defaultMessage: 'Documents (%)',
         }),
         render: (value: number | undefined, item: DataVisualizerTableItem) => (
-          <DocumentStat config={item} showIcon={dimensions.showIcon} />
+          <DocumentStat config={item} showIcon={dimensions.showIcon} totalCount={totalCount} />
```

Comment:
For a data view without a time field, you get `Infinity %` for percentages in the document stats and top values:

<img width="497" alt="image" src="https://user-images.githubusercontent.com/7405507/186693845-6883a1ec-e7cc-4cd9-8b63-593dbad6c8bc.png">